status/clusters_per_lane.py

```diff
@@ -0,0 +1,78 @@
+import tornado.web
+import json
+import cStringIO
+
+import matplotlib.gridspec as gridspec
+from matplotlib.figure import Figure
+from matplotlib.backends.backend_agg import FigureCanvasAgg
+
+from status.util import SafeHandler
+
+class ClustersPerLaneHandler(SafeHandler):
+    """ Serves a page with a plot of distribution of lane read production for a provided
+    time interval.
+    """
+    def get(self):
+        names= self.application.flowcells_db.view("lanes/clusters")
+        start=names.rows[0].key[0][:6]
+        end=names.rows[-1].key[0][:6]
+        start=start[:4]+'01'
+        end=end[:4]+'31'
+        t = self.application.loader.load("clusters_per_lane.html")
+        self.write(t.generate(user = self.get_current_user_name(), start=start, end=end, deprecated=True))
```

Comment:
God I'm so glad we invented the internet in America and got to pick all the spellings